scripts/download_backtest_data.py

```diff
@@ -4,8 +4,8 @@
 import json
 import sys
 import os
-
-from freqtrade import (exchange, arguments, misc)
```

Comment:
I think you'll want to keep this import, otherwise `exchange.` in line 34 and `arguments.` in line 12 will not be available.